.stylelintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "stylelint-config-standard-scss",
+  "rules": {
+    "no-descending-specificity": null
+  },
+  "ignoreFiles": [
+    "src/**/scss/plugins/*.scss",
+    "src/Themes/Fork/assets/webpack/scss/components/_buttons.scss"
```

Comment:
Waarom staat die bij de ignored files?

Reply:
Die _buttons.scss is op dit moment leeg, en dit geeft errors in de ci pipeline

Reply:
Ja die mag gewoon weg

Reply:
fixed in 7c4eb714e427c0ad55df7c7a69142dc8a9b918e1